frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelStepsTable.tsx

```diff
@@ -92,8 +93,23 @@ export function FunnelStepsTable(): JSX.Element | null {
                 },
                 {
                     title: 'Total conversion',
-                    render: (_: void, breakdown: FlattenedFunnelStepByBreakdown) =>
-                        formatDisplayPercentage(breakdown?.conversionRates?.total ?? 0, true),
+                    render: function RenderTotalConversion(
```

Comment:
curious about when and where does 'Total conversion' show up? Not sure I ever see this. Otherwise LGTM!

Reply:
D'oh, ofcourse. Gahh, let me go again :P